packages/daimo-userop/src/nonce.ts

```diff
@@ -77,3 +76,7 @@ export class DaimoNonceMetadata {
     return (hexNonceType + hexIdentifier) as Hex;
   }
 }
+
+function assert(condition: boolean) {
+  if (!condition) throw new Error("Assertion failed");
```

Comment:
avoids a circular dependency.

`daimo-userop` only depends on `daimo-contracts` no, and both can be published to npm